Automattic/Edit-Flow
# Add support for Slack notifications

## Description

This PR adds support for Slack notifications on top of EditFlow's existing email notifications. Currently we're only planning to support Slack's incoming webhook API.

## Steps to Test

You would need to create a Slack app on your own workspace and enable Incoming Webhooks for that app.

1. Check out PR.
1. Go to Notifications.
1. Enable `Send to Webhook` and add in the webhook URL.
1. Save settings
1. Change a post status or add editorial comments.
1. Verify that Slack messages were sent to the right channel on your Slack workspace
